quantmsrescore/snr.py

```
@@ -1,227 +1,297 @@
+
+                psm_metavalues.append({
+                    "name": feature,
+                    "value": str(value),
+                    "value_type": value_type
+                })
+                added_features.add(feature)
+
+            record["psm_metavalues"] = psm_metavalues
+            result_rows.append(record)
+
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)